docs/DeveloperGuide.md

```diff
@@ -407,12 +418,13 @@ A short description of the process including the parsing of macros is as follows
 1. If command word is `macro`, return a `NewMacroCommand` (which will create a new `macro` when executed).
 2. Else if command word is an existing macro, call the `asCommandInstance` method in the `MacroRunner` class to create a `CommandExecutable` from the macro and return that.
 3. Else hand over to primitive command parser.
+4. Else, a CommandException will be thrown
```

Comment:
A repeated Else statement here, maybe otherwise sounds better?

Reply:
I'll change it, should also be parseerror